frontend/src/components/Home.js

```diff
@@ -28,13 +44,13 @@ export class Home extends Component {
     return (
       <Layout className="layout">
         <div className="header">
-          <NavBar />
+          <NavBar activeButton={this.state.activeNavButton} />
           <LoadingBar
             style={{
-              backgroundColor: "#6b6363",
+              backgroundColor: "#5e46a1",
```

Comment:
Bringing this up again. Constants for JSX colours?

Reply:
yeah I'll make that change!